sycl/test-e2e/Matrix/Inputs/joint_matrix_out_bounds_impl.hpp

```diff
@@ -147,13 +148,38 @@ void test() {
 
 template <layout A_layout, layout B_layout> void test_all() {
   std::cout << "bf16: ";
-  test<bfloat16, float, /*MATRIX_M*/ 1024 + 20, /*MATRIX_N*/ 1024 + 20,
+  test<bfloat16, float, /*MATRIX_M*/ 1024 + 24, /*MATRIX_N*/ 1024 + 24,
```

Comment:
Size update is due to this requirement: For Untyped 2D Block Messages, Surface Pitch must be a multiple of OW(16B).